crates/services/executor/src/executor.rs

```diff
@@ -1626,7 +1626,8 @@ where
             .iter()
             .map(|input| input.predicate_gas_used())
             .collect();
-        let ready_tx = checked_tx.into_ready(gas_price, gas_costs, fee_params)?;
+        // TODO: Add support for expiration policy
```

Comment:
I beileve this should be `Some(*header.block_height())` but please check @AurelienFT

Reply:
Yes I will add it.